samples/excel/worksheet/hide-unhide-worksheet.yaml

```diff
@@ -0,0 +1,168 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Hide and unhide worksheet
+description: Explore the APIs to hide and unhide a worksheet
+script:
+  content: >-
+    $("#hide-worksheet").click(hideWorksheet);
+
+    $("#unhide-worksheet").click(unhideWorksheet);
+
+
+    async function hideWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+
+                let visibleSheets = await filterWorksheetsByVisibility(context, Excel.SheetVisibility.visible);
+
+                if (visibleSheets.length > 1) {
+                    console.log(`Hiding worksheet named "${visibleSheets[0].name}"...`);
+
+                    visibleSheets[0].visibility = Excel.SheetVisibility.hidden;
+
+                    await context.sync();
+
+                } else {
+                    console.log("Unable to hide the last visible worksheet in the workbook");
```

Comment:
Also - if there are sheets that are not worksheets, then this will work, e.g. if there's a ChartSheet (which is not a worksheet), then you can hide the last worksheet since there's still a sheet available.